pkg/setup/generate_bundle_stub.go

```diff
@@ -0,0 +1,14 @@
+//+build !withbundle
+
+package setup
+
+//go:generate go-bindata -pkg setup -o bundled_plugins_stub_linux.gen.go -ignore=\.gitignore -nometadata -dev -tags "!withbundle" -prefix "bundled_plugins" bundled_plugins/linux/
+//go:generate go-bindata -pkg setup -o bundled_plugins_stub_darwin.gen.go -ignore=\.gitignore -nometadata -dev -tags "!withbundle" -prefix "bundled_plugins" bundled_plugins/darwin/
+//go:generate go-bindata -pkg setup -o bundled_plugins_stub_windows.gen.go -ignore=\.gitignore -nometadata -dev -tags "!withbundle" -prefix "bundled_plugins" bundled_plugins/windows/
```

Comment:
I don't understand why we need to generate stuff here, as go-bindata relies on an `Asset` function, couldn't this file only define this "Asset" function and it'd return an error like `no core CLI is bundled in the binary`?

Reply:
Doing that gives us the ability to actually run the code and test this without needing to generate 3 real bundles and recompile. I suppose taking it out might be an improvement but then I think we should change our makefile to at least make it possible to easily generate a bundle without needing to use `GO_BUILD_TAGS` manually.